.github/workflows/deploy_dev_stage_prod.yml

```
@@ -0,0 +1,128 @@
+name: Deploy to Dev, Stage, and Prod
+
+on:
+  workflow_call:
+    inputs:
+      commit-identifier:
+        description: SHA or tag to apply, (optional) when omitted, the head of the selected branch is used
+        type: string
+        required: true
+      terraform-workspace:
+        description: Terraform workspace that is used
+        type: string
+        required: true
+      terraform-root: 
+        description: Root directory of terraform, default to 'terraform/${ inputs.environment }'
+        type: string
+        required: false
+    secrets: 
+      ORG_READ_ONLY_SSH_KEY: 
+        required: true
+      ORG_GITHUB_PACKAGES_READ_ONLY_TOKEN:
+        required: true
+      CORE_ENG_DEPLOYMENTS_SLACK_WEBHOOK:
+        required: true
+
+concurrency: ${{ github.workflow }}-deploy_dev_stage_prod
+
+permissions:
+  id-token: write
+  contents: write
+  pull-requests: write
+
+env:
+  TERRAFORM_WORKSPACE: ${{ inputs.terraform-workspace }}
+
+jobs:
+  Setup:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.merged }}
+    outputs:
+      release-tag: ${{ steps.releaseTag.outputs.release-tag }}
+
+    steps:
+      - name: Set Release Tag
+        id: releaseTag
+        run: echo "release-tag=$(date +'%Y.%m.%d-%H-%M')" >> $GITHUB_OUTPUT
+
+  DeployDev:
+    name: Apply Dev
+    needs: [Setup]
+    uses:  opensesame/core-github-actions/.github/workflows/deploy_environment.yml
```

since these workflows are defined in this repo you can probably reference it relatively? 
```suggestion
    uses:  .github/workflows/deploy_environment.yml
```

---

Yea wasn't sure if it was relative, but let's go with that for now